invenio_app_rdm/theme/static/robots.txt

```diff
@@ -2,4 +2,11 @@ User-agent: *
 Disallow: /search
 Disallow: /api
 Disallow: /administration
+Disallow: /account
+Disallow: /uploads
+Disallow: /me
```

Comment:
These are generally only available when the user is authenticated:
* `/account/*`: profile settings, tokens, etc.
* `/me/*`: the user dashboard
* `/uploads/*`: the deposit form